frontend/src/services/ApiService.js

```
@@ -0,0 +1,252 @@
+      // Something else happened
+      return new Error(`Request error: ${error.message}`);
+    }
+  }
+
+  // Health check
+  async checkHealth() {
+    try {
+      const response = await this.client.get('/health');
+      return response.data;
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)